articles/custom-domains/_provide-domain-name.md

```diff
@@ -0,0 +1,21 @@
+## Provide your domain name to Auth0
+
+1. Go to [Dashboard > Tenant Settings](${manage_url}/#/tenant). 
+
+2. Select the **Custom Domains** tab.
+
+  <% if (platform === "auth0") { %>
```

Comment:
If a customer has multiple domains enabled, what happens in this situation, @tamigoodall ?

Reply:
I believe they can only have one domain per tenant. I will find out what happens if they try to add another.

Reply:
We allow it in special cases and are looking at adding it in 2020. I can send you a tenant with a second domain if you need one for testing.

Reply:
They receive an error message.

Reply:
Is this going to be a problem? How can we support their use case? They are usually large enterprise customers